Source/Classes/AnimatedImages/PINAnimatedImage.m

```diff
@@ -0,0 +1,101 @@
+//
+//  PINAnimatedImage.m
+//  PINRemoteImage
+//
+//  Created by Garrett Moon on 9/17/17.
+//  Copyright © 2017 Pinterest. All rights reserved.
+//
+
+#import "PINAnimatedImage.h"
+
+NSString *kPINAnimatedImageErrorDomain = @"kPINAnimatedImageErrorDomain";
+
+const NSTimeInterval kPINAnimatedImageDisplayRefreshRate = 60.0;
```

Comment:
This value isn't true on some newer devices (the new iPad Pros). Could this instead be gleaned from `UIDevice` somehow?